code/modules/awaymissions/mission_code/ruins/deepstorage.dm

```diff
@@ -0,0 +1,382 @@
+// When reviewing this, please consider someone who didn't know what they were doing just copy-pasted a lot of things
```

Comment:
```suggestion
```
Reviews are a great place to learn! No worries

I'll make sure to annotate my reviews a bit better so you know why I'm requesting certain things.